src/stores/database.js

```diff
@@ -115,6 +116,14 @@ class Database {
     await umzug.up();
   }
 
+  async generateSchemaHash() {
```

Comment:
Since we generate this only once I think there is a very small chance that this can be outdated (if the next version already runs the migrations behind, but is not serving yet), but it's so small that I guess we don't need to worry about it.

Reply:
We have already the problem of the DB being migrated but the corresponding code not being served yet, which can cause problems all over the project. So yeah, nothing new or particular to this PR.